raft/src/main/java/org/apache/kafka/raft/RaftUtil.java

```diff
@@ -16,24 +16,30 @@
  */
 package org.apache.kafka.raft;
 
+import org.apache.kafka.common.TopicPartition;
 import org.apache.kafka.common.message.BeginQuorumEpochResponseData;
+import org.apache.kafka.common.message.DescribeQuorumRequestData;
 import org.apache.kafka.common.message.EndQuorumEpochResponseData;
 import org.apache.kafka.common.message.FetchQuorumRecordsResponseData;
 import org.apache.kafka.common.message.FindQuorumResponseData;
+import org.apache.kafka.common.message.VoteRequestData;
 import org.apache.kafka.common.message.VoteResponseData;
 import org.apache.kafka.common.protocol.ApiKeys;
 import org.apache.kafka.common.protocol.ApiMessage;
 import org.apache.kafka.common.protocol.Errors;
 import org.apache.kafka.common.record.FileRecords;
 import org.apache.kafka.common.record.MemoryRecords;
 import org.apache.kafka.common.record.Records;
+import org.apache.kafka.common.requests.VoteResponse;
 
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.util.OptionalInt;
 
 public class RaftUtil {
 
+    static final TopicPartition METADATA_PARTITION = new TopicPartition("metadata", 0);
```

Comment:
I think we need to be consistent on whether this topic is hard-coded or not. Currently `KafkaRaftClient` uses the value from `ReplicatedLog.topicPartition()`. Do we want to make `ReplicatedLog` the source of truth for this value?

Reply:
This hard-coded value is only used in tests, let me move it.